novawallet/Common/Services/ChainRegistry/ChainFilterStrategy.swift

```diff
@@ -25,14 +25,24 @@ enum ChainFilterStrategy {
                 }
             }
         case .hasProxy: { change in
+                if case .delete = change {
+                    return true
+                }
+
                 #if F_RELEASE
                     return change.item?.hasProxy == true
                         && change.item?.isTestnet == false
                 #else
                     return change.item?.hasProxy == true
                 #endif
             }
-        case let .chainId(chainId): { $0.item?.chainId == chainId }
+        case let .chainId(chainId): { change in
+            if case .delete = change {
```

Comment:
I think we need to compare chainId with deleteIdentifier

Reply:
Right